README.md

```diff
@@ -159,7 +160,25 @@ Options:
                                   var: LOGILICA_EMAIL]
   --help                          Show this message and exit.
 ```
+
 Some dashboards can be quite slow to load into the UI, so, if you are running
 the tool interactively and wish to track its progress, add a `-v` to the command
-line to see informational messages.  (If you are debugging or just nosy, adding
+line to see informational messages. (If you are debugging or just nosy, adding
 a _second_ `-v` will add debugging messages to the output.)
+
+## Development
+
+### Linting and formatting rules
+
+This project uses `pre-commit` to handle linting and formatting. Use the following steps to be able to run it locally:
+
+```
+pip install pre-commit
+pre-commit install
+pre-commit run --all-files --hook-stage [pre-commit|manual]
+```
+
+If you run `pre-commit` stage, you can check your files, if you run `manual` stage, it will actually lint and format your files.
+
```

Comment:
I would replace "it will actually lint and format your files" with something stronger, like "it will modify your files to correct problems it finds."